app/Http/Controllers/API/AuthController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Services\UserService;
+use App\Http\Requests\UserRequest;
+use App\Http\Controllers\Controller;
+use App\Http\Resources\UserResource;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class AuthController extends Controller
+{
+    protected $userService;
+
+    public function __construct(UserService $userService)
+    {
+        $this->userService = $userService;
+    }
+
+    public function login(UserRequest $request): JsonResponse
```

Comment:
@sabinshrestha8 I see you throwing exception for invalid credentials, have you handled that in the generic exception handler?